iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_queue_data.hpp

```diff
@@ -26,24 +26,24 @@ namespace iox
 {
 namespace popo
 {
+template <typename ChunkQueueDataProperties>
 struct ChunkQueueData
 {
+    using ChunkQueueDataProperties_t = ChunkQueueDataProperties;
+
     explicit ChunkQueueData(cxx::VariantQueueTypes queueType) noexcept;
 
-    static constexpr uint32_t MAX_CAPACITY = MAX_RECEIVER_QUEUE_CAPACITY;
+    static constexpr uint32_t MAX_CAPACITY = ChunkQueueDataProperties_t::MAX_QUEUES;
```

Comment:
Maybe this is nor more needed if we could now use ChunkQueueData<..>::ChunkQueueDataProperties_t ::MAX_QUEUES? 

On the other hand, this one more line will make others more readable...

Reply:
That's what I had in my mind too. Either way is fine for me. What about the others?

Reply:
Fine for me as it is